docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html

```diff
@@ -1,8 +1,15 @@
-<label for="file-upload-demo" ng2FileDrop [class.file-upload-dropping]="fileOver" (fileOver)="fileOver = $event" [uploader]="uploader" class="file-upload-dropzone">
+<label for="file-upload-demo"
+    tabindex="0"
+    ng2FileDrop
+    [class.file-upload-dropping]="fileOver"
+    (fileOver)="fileOver = $event"
+    [uploader]="uploader"
+    class="file-upload-dropzone"
+    (keydown.enter)="upload.click()">
     Click here or Drag and Drop to upload files
 </label>
 
-<input type="file" id="file-upload-demo" class="sr-only" ng2FileSelect [uploader]="uploader" multiple>
+<input #upload type="file" id="file-upload-demo" class="sr-only" ng2FileSelect [uploader]="uploader" multiple>
```

Comment:
This is receiving tab focus after the label above, so it should probably be removed from the tab ring.